Stripe/STPPaymentMethodsInternalViewController.m

```diff
@@ -145,6 +145,16 @@ - (CGFloat)tableView:(__unused UITableView *)tableView heightForHeaderInSection:
     return 0.01f;
 }
 
+- (void)updateWithPaymentMethodTuple:(STPPaymentMethodTuple *)tuple {
+    if (self.paymentMethods == tuple.paymentMethods && self.selectedPaymentMethod == tuple.selectedPaymentMethod) {
```

Comment:
Should this be `isEqual` or are you intentionally checking for pointer comparison?